src/components/lists/items/CallAssignmentListItem.jsx

```diff
@@ -41,48 +44,104 @@ export default class CallAssignmentListItem extends React.Component {
 
     render() {
         let assignment = this.props.data;
+        let progressSum = 0;
+        let callsStats = null;
+        let reachedStats = null;
         let targetStats = null;
         let goalStats = null;
-        let participantList = null;
+        let participantIndicator = null;
+        const assignmentDateStart = new Date(assignment.start_date);
+        const assignmentDateEnd = new Date(assignment.end_date);
+        const inPast = (assignmentDateEnd < (new Date()) ? true : false);
+
+        const classNames = cx({
+            'CallAssignmentListItem': true,
+            'past': inPast
+        });
+
+        const assignmentDateSpan =
```

Comment:
I think we need to include year in this. I have some really long-running assignments in my dummy data, and was confused by the date output because I didn't first realize that one assignment started in 2016 and ended at the end of 2017.